ote_sdk/ote_sdk/entities/dataset_item.py

```diff
@@ -247,26 +247,30 @@ def get_annotations(
         self,
         labels: Optional[List[LabelEntity]] = None,
         include_empty: bool = False,
+        include_ignored: bool = False,
     ) -> List[Annotation]:
         """
         Returns a list of annotations that exist in the dataset item (wrt. ROI). This is done by checking that the
         center of the annotation is located in the ROI.
 
         :param labels: Subset of input labels to filter with; if ``None``, all the shapes within the ROI are returned
         :param include_empty: if True, returns both empty and non-empty labels
+        :param include_ignored: if True, includes the labels in ignored_labels
         :return: The intersection of the input label set and those present within the ROI
         """
         is_full_box = Rectangle.is_full_box(self.roi.shape)
         annotations = []
-        if is_full_box and labels is None and not include_empty:
+        if is_full_box and labels is None and not include_empty and not include_ignored:
```

Comment:
Hm... Should not that be `and include_empty and include_ignored`?

Reply:
I think you're right. The ticket referenced below also mentions it https://jira.devtools.intel.com/browse/CVS-75919. I believe this will only be seen in multi class classification because of the full box constraint.